javascripts/discourse/initializers/initialize-for-header-icon-links.gjs

```diff
@@ -30,32 +24,31 @@ export default {
         splitLinks.forEach((link, index, links) => {
           const fragments = link.split(",").map((fragment) => fragment.trim());
           const title = fragments[0];
-          const icon = buildIcon(fragments[1]);
+          const iconTemplate = buildIcon(fragments[1], title);
           const href = fragments[2];
           const className = `header-icon-${dasherize(fragments[0])}`;
           const viewClass = fragments[3].toLowerCase();
           const target = fragments[4].toLowerCase() === "blank" ? "_blank" : "";
           const rel = target ? "noopener" : "";
           const isLastLink =
             link === links[links.length - 1] ? ".last-custom-icon" : "";
```

Comment:
Don't think we want the preceding `.` here?
```suggestion
            link === links[links.length - 1] ? "last-custom-icon" : "";
```

Reply:
Good catch, thanks!